src/components/score-ribbon/index.tsx

```diff
@@ -1,6 +1,43 @@
 import { Col, Row } from 'antd'
+import 'antd/dist/antd.css'
+import { ColProps } from 'antd/lib/col'
 import * as React from 'react'
-import './index.css'
+import styled from 'styled-components'
+
+interface ITeam {
+  team: 'ct' | 't'
+}
+
+interface IScoreProps extends ColProps, ITeam {}
+interface INameProps extends ColProps, ITeam {}
+
+const CT = () => '#0079b9'
+const T = () => '#e4c273'
+
+const Rounds = styled<ColProps>((props) => <Col {...props} />)`
+  text-align: center;
+`
+
+const Time = styled<ColProps>((props) => <Col {...props} />)`
+  font-weight: bold;
+  font-size: 1rem;
+  text-align: center;
+`
+
+const Score = styled<IScoreProps>((props) => <Col {...props} />)`
```

Comment:
Here is an example where I wrap an `antd` component in styled-components. Maybe this can be done in a better way?

Reply:
You don't need to construct a new component (the arrow function you've written). Just do 

```ts
styled<IScoreProps>(Col)`
  font-size: 100rem;
`;
```

Reply:
Like this?:

```ts
const Score = styled<IScoreProps>(Col)`
  text-align: center;
  font-weight: bold;
  font-size: 1.2rem;
  color: ${(props) => (props.team === 'ct' ? CT : T)};
`
```

This gives me two errors:

- Argument of type 'typeof Col' is not assignable to parameter of type 'ComponentType<IScoreProps>'.
- Parameter 'props' implicitly has an 'any' type.